general-information.md

```diff
@@ -293,56 +314,29 @@ MDS uses Version 1 UUIDs by default. Version 4 UUIDs may be used where noted.
 
-### Event Types
-
-Event types are the possible transitions between some vehicle states.  
-
-| `event_type` | Description |
-|--------------|-------------|
-| `agency_drop_off` |	Drop off by the agency	|
-| `agency_pick_up` |	Pick up by the agency	|
-| `battery_charged` |	Battery charged	|
-| `battery_low` |	Battery low	|
-| `comms_lost` |	Communications lost	|
-| `comms_restored` |	Communications restored	|
-| `compliance_pick_up` |	Pick up for compliance	|
-| `decommissioned` |	Decommissioned	|
-| `located` |	Located	|
-| `maintenance` |	General maintenance	|
-| `maintenance_pick_up` |	Pick up for maintenance	|
-| `missing` |	Missing	|
-| `off_hours` |	Off hours - end of service	|
-| `on_hours` |	On hours - start of service	|
-| `provider_drop_off`	 |	Drop off by the provider	|
-| `rebalance_pick_up` |	Pick up for rebalancing	|
-| `reservation_cancel` |	Reservation cancelled	|
-| `reservation_start` |	Reservation started	|
-| `system_resume` |	Resume system operations	|
-| `system_suspend`	 |	Suspend system operations	|
-| `trip_cancel` |	Cancel trip	|
-| `trip_end` |	End trip	|
-| `trip_enter_jurisdiction` |	Trip enters a jurisdiction	|
-| `trip_leave_jurisdiction` |	Trip leaves a jurisdiction	|
-| `trip_start` |	Start trip	|
-| `unspecified` |	Unspecified	|
+| `vehicle_state`   | In PROW? | Description                                                                                                                                                                    |
+|-------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `removed`         | no       | Examples include: at the Provider's warehouse, in a Provider's truck, or destroyed and in a landfill.                                                                          |
+| `available`       | yes      | Available for rental via the Provider's app. In PROW.                                                                                                                          |
+| `non_operational` | yes      | Not available for rent.  Examples include: vehicle has low battery, or currently outside legal operating hours.                                                                |
+| `reserved`        | yes      | Reserved via Provider's app, waiting to be picked up by a rider.                                                                                                               |
+| `on_trip`         | yes      | In possession of renter.  May or may not be in motion.                                                                                                                         |
+| `elsewhere`       | no       | Outside of regulator's jurisdiction, and thus not subject to cap-counts or other regulations. Example: a vehicle that started a trip in L.A. has transitioned to Santa Monica. |
```

Comment:
From WG call. Might need to rethink this as edge cases come up and at least define the cases and how elsewhere is used per mode. Crossing jurisdictions in Agency with ids is difficult. Maybe eliminate location data from elsewhere? Concept of not in any jurisdiction, even someone else's.